tests/test_parser_v2.py

```diff
@@ -76,7 +76,7 @@ def bytes_to_service_info(
         rssi=-60,
         manufacturer_data={},
         service_data={"0000fcd2-0000-1000-8000-00805f9b34fb": payload},
-        service_uuids=["0000fcd2-0000-1000-8000-00805f9b34fb"],
+        service_uuids=[],
```

Comment:
The button doesn't send this in passive mode and the tests pass without it so I took it out